src/Glpi/Form/Destination/CommonITILField/ContentField.php

```diff
@@ -120,6 +121,13 @@ public function applyConfiguratedValueToInputUsingAnswers(
             $answers_set
         );
 
+        // Add margin to h2 tags to improve readability
+        $input['content'] = preg_replace(
+            '/<h2>/',
+            '<h2 style="margin: 1rem 0 0.5rem;">',
```

Comment:
IMHO, if some styles could be improved, then they should be improved on every `.rich_text_container` containers.